bin/node/src/aleph_cli.rs

```diff
@@ -37,4 +40,18 @@ impl AlephCli {
     pub fn backup_path(&self) -> Option<PathBuf> {
         self.backup_path.clone()
     }
+
+    pub fn insert_default_backup_path(&mut self, config: &Configuration) {
```

Comment:
Let's use the path itself as a parameter instead of the whole config

Reply:
Done